stable/guardian/values.yaml

```diff
@@ -85,6 +101,8 @@ app:
     JOBS_REVOKE_EXPIRED_ACCESS_INTERVAL: "*/20 * * * *"
     JOBS_EXPIRING_ACCESS_NOTIFICATION_ENABLED: false
     JOBS_EXPIRING_ACCESS_NOTIFICATION_INTERVAL: "0 9 * * *"
```

Comment:
can we add grant related flags also here?

Reply:
those are optional, we can omit them from here

Reply:
Sure. Some of these existing ones are deprecated if I recall correctly. Can we remove the deprecated once also.